extensions/amp-form/0.1/amp-form.js

```diff
@@ -183,22 +186,24 @@ export class AmpForm {
       return;
     }
 
-    // Validity checking should always occur, novalidate only circumvent
-    // reporting and blocking submission on non-valid forms.
-    const isValid = checkUserValidityOnSubmission(this.form_);
-    if (this.shouldValidate_ && !isValid) {
-      if (opt_event) {
-        opt_event.stopImmediatePropagation();
-        opt_event.preventDefault();
+    if (isCheckValiditySupported(this.win_.document)) {
+      // Validity checking should always occur, novalidate only circumvent
+      // reporting and blocking submission on non-valid forms.
+      const isValid = checkUserValidityOnSubmission(this.form_);
+      if (this.shouldValidate_ && !isValid) {
```

Comment:
If `checkValidaty` is not supported, should we set `isValid` to false, instead of skipping all the operations for not valid form?